pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -153,3 +142,47 @@ func getGithubReleases() ([]string, error) {
 	}
 	return releaseTags, nil
 }
+
+func GetSortedOpenShiftTags(minSupportedVersion, defaultVersion string) ([]string, error) {
+	var releaseList []string
+
+	data, err := getGithubReleases()
+	if err != nil {
+		return []string{}, err
+	}
+
+	for _, releaseTag := range data {
+		if valid, _ := IsGreaterOrEqualToBaseVersion(releaseTag, minSupportedVersion); valid {
+			if valid, _ := IsGreaterOrEqualToBaseVersion(releaseTag, defaultVersion); valid {
```

Comment:
@budhram Just question, how it comes here that v3.11.0-alpha.0 and v3.8.0 are not appended?

Reply:
Aaah, GetGithubReleases() does not return those, need to check that.

Reply:
Ya @agajdosi and `v3.11.0-alpha.0` is not a proper release with its artifacts. See https://github.com/openshift/origin/releases

Reply:
@budhram Haha! Completely missed that, was still searching where do we have an exception for those versions, thanks, it is clear now!